cmd/agentctl/main.go

```diff
@@ -79,6 +81,45 @@ source-of-truth directory.`,
 	return cmd
 }
 
+func samplesCmd() *cobra.Command {
+	var selector string
+
+	cmd := &cobra.Command{
+		Use:   "samples [WAL directory]",
+		Short: "Discover sample statistics for series matching a label selector within the WAL.",
```

Comment:
Can you give this a `Long` documentation with an example of a value for the selector flag? It's not immediately clear to me how it could be used.

Reply:
Done that, also ensured the default selector is matching all series